branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/IBranchLayout.java

```diff
@@ -46,4 +46,6 @@ default boolean equals(IBranchLayout other) {
 
     return false;
   }
+
+  List<IBranchLayoutEntry> getEntriesOrderedList();
```

Comment:
I am wondering if we really need to expose this one. 
Now we use it only in a single place (TraverseRepositoryAction) - we could simply implement this method there. 

I am assuming that we want to keep api as simple as possible